packages/block-editor/src/components/block-styles/style.scss

```diff
@@ -43,6 +43,16 @@
 		display: inline-block;
 		width: calc(50% - #{$grid-unit-05});
 
+		&-text {
```

Comment:
Nit: I'd prefer this to be written out (`.block-editor-block-styles__item-text`) so it can be searched easily across the codebase.

Also, no need to nest it this deep since the class will never appear anywhere else, and it just adds unnecessary specificity.

Reply:
@mirka 
Thanks for the review.

I've adjusted the `.block-editor-block-styles__item-text` class to make it easier to isolate and search.

How about something like this?